docs/guides/references/_changelogs/12.4.0.md

```diff
@@ -0,0 +1,63 @@
+## 12.4.0
+
+_Released 1/24/2023_
+
+**Features:**
+
+- Added official support for Vite 4 projects for component testing. Addresses
```

Comment:
```suggestion
- Added official support for Vite 4 in component testing. Addresses
```

@ZachJW34 Do we have vite documenation in the docs? Not finding it here https://docs.cypress.io/guides/core-concepts/cypress-app#Selector-Playground

Reply:
or is this vite-dev-server?

Reply:
addressed in 762c22139